spot-contracts/contracts/BondIssuer.sol

```diff
@@ -140,17 +180,29 @@ contract BondIssuer is IBondIssuer, OwnableUpgradeable {
     /// @dev Lazily issues a new bond when the time is right.
     function getLatestBond() external override returns (IBondController) {
         issue();
-        // NOTE: The latest bond will be at the end of the list.
-        return IBondController(_issuedBonds.at(_issuedBonds.length() - 1));
+        return _lastBond;
```

Comment:
Is the purpose of adding the new contract storage variable to optimize this function call?

This adds more gas cost to the `issue()` function and therefore `updateState()` on Perp.

Reply:
yea makes sense. Maybe we could just return the last item in `_activeBonds` ?

Reply:
We need to keep track of the last issued bond somehow. We can't rely on fetching the last element in `_activeBonds` as `_activeBonds` could be empty (if all the bonds mature).  `_maturedBonds` on the other hand will be ordered by maturity date and not by issue date (in practice though it should be ordered by issue date but its not guaranteed).

Reply:
This does add an an additional storage update on "issue" but it's a onetime cost.